src/locales/scripts/types.d.ts

```diff
@@ -1,3 +1,9 @@
-/**
- * @typedef {{ code: string, wpLocale?: string, file?: string, iso?: string, name: string, dir?: string, translated?: number }} i18nLocaleProps
- */
+interface i18nLocaleProps {
```

Comment:
Nit (and not even really part of the changes you made but would be good to change if we can): Using uppercase letters to start interfaces and types is considered good practice as a way to easily distinguish them from instances.

We can also use just a regular `.ts` file for this rather than a `.d.ts`. It's also necessary to add an `export` to this declaration (JSDoc `@typedef` types are implicitly exported where as in regular TypeScript you have to be explicit about which types to export).

```suggestion
export interface I18nLocaleProps {
```

Reply:
Fixed in [16cd2c9](https://github.com/WordPress/openverse-frontend/pull/1023/commits/16cd2c9d63c064e515d50842fe2912a9195c7b07).